src/lib.rs

```diff
@@ -99,20 +97,20 @@ impl ZipfDistribution {
         // looks slightly different.
 
         // We know these were computed in new()
-        let hnum = self.h_integral_num_elements.unwrap();
-        let h_x1 = self.h_integral_x1.unwrap();
-        let s = self.s.unwrap();
+        let hnum = self.h_integral_num_elements;
+        let h_x1 = self.h_integral_x1;
+        let s = self.s;
```

Comment:
Now that these are no longer `Option`, we can just use them inline below instead :)